app/main.py

```diff
@@ -1 +1,12 @@
-# write your code here
+def copy_file(command: str) -> None:
+    parts = command.split()
+
+    if len(parts) == 3 and parts[0] == "cp":
+        source_file, target_file = parts[1], parts[2]
+
+        if source_file == target_file:
+            return
+
+        with (open(source_file, "r") as file_in,
+              open(target_file, "w") as file_out):
```

Comment:
Consider adding error handling for file operations. If the source file does not exist or cannot be opened, the program will raise an exception. You can use a try-except block to handle such cases gracefully.